recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "optuna-integration" %}
-{% set version = "3.5.0" %}
+{% set version = "3.6.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 18f24838a63d8f3b733f184ad59dfbfe4a7959401d3c9ac84f307134b8231893
+  sha256: f261c38586b22cd95639287ca694fc0f788482cfbb7bb83803caf404ce06a55a
```

Comment:
This is consistent with https://pypi.org/project/optuna-integration/#copy-hash-modal-0a23aeeb-1a2e-4e74-b2e7-0d9fafd5610b